src/events/aws_sdk.js

```diff
@@ -923,9 +926,35 @@ const SSMEventCreator = {
         switch (response.request.operation) {
         case 'getParameter':
             eventInterface.addToMetadata(event, {
-                value: (response.data.Parameter || { Value: '' }).Value || '',
+                parameter: {
+                    Name: (response.data.Parameter || { Name: '' }).Name || '',
+                    Value: (response.data.Parameter || { Value: '' }).Value || '',
+                    Type: (response.data.Parameter || { Type: '' }).Type || '',
+                },
             });
             break;
+
+        case 'getParameters':
+        case 'getParametersByPath':
+            if (response.data.InvalidParameters && response.data.InvalidParameters.length > 0) {
+                eventInterface.addToMetadata(
+                    event,
+                    { invalid_parameters: response.data.InvalidParameters }
+                );
+            }
+
+            if (response.data.Parameters && response.data.Parameters.length > 0) {
+                eventInterface.addToMetadata(event, {
+                    parameters: response.data.Parameters
+                        .map(singleParameter => ({
+                            Path: singleParameter.Path,
+                            Value: singleParameter.Name,
+                            Type: singleParameter.Type,
```

Comment:
are we sure we want to capture the parameter value? I think it should be a secret

Reply:
@ranrib I think you faced this question before

Reply:
That should go to the non metadata section

Reply:
fixed